node/remote/node.go

```diff
@@ -39,7 +41,8 @@ type Node struct {
 
 // NewNode allows to build a new Node instance
 func NewNode(cfg *Details) (*Node, error) {
-	httpClient, err := jsonrpcclient.DefaultHTTPClient(cfg.RPC.Address)
+	rpcAddress := utils.GetEnvOr(env.RPCAddress, cfg.RPC.Address)
```

Comment:
Is it difficult to set the config RPC from the env where the cfg is created?

Reply:
It's a standard k8s pod so it it easy to do